cli/command/stack/remove.go

```diff
@@ -39,7 +39,17 @@ func runRemove(dockerCli command.Cli, opts removeOptions) error {
 	client := dockerCli.Client()
 	ctx := context.Background()
 
-	var errs []string
+	var (
+		errs    []string
+		secrets []swarm.Secret
+		configs []swarm.Config
+	)
```

Comment:
This is a minor nit, but I think it's generally better to keep variable declarations closer to the code where they are used. It makes it easier to see the scope of the variable and to refactor the code later.

So these could move to immediately before the `if versions...` check for each